master/static/migrations/20230725094922_add-rp-workspace-bindings-constraint.tx.down.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE rp_workspace_bindings
+    DROP CONSTRAINT rp_workspace_pkey;
```

Comment:
Nit: all lines should have the POSIX-compliant terminating `\n`, including the final lines of files.